test/Project.toml

```diff
@@ -4,6 +4,7 @@ DimensionalData = "0703355e-b756-11e9-17c0-8b28908087d0"
 Distances = "b4f34e82-e78d-54a5-968a-f98e89d6e8f7"
 Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
 DynamicalSystemsBase = "6e36e845-645a-534a-86f2-f5d4aa5a06b4"
+Integrals = "de52edbc-65ea-441a-8357-d3a637375a31"
```

Comment:
There was a comment in the `tranfer_operator.jl` test file:

```julia
# There's not easy way of constructing an analytical example for the resulting
# probabilities, because they are computed according to the approximation to the
# transfer operator, not as naive visitation freqiencies to the bins. We just test
# generically here.
# TODO: make a stupidly simple example where we can actually compute the measure of
# each bin exactly.
```
So I've added a test for the logistic map (r=4) where the  invariant measure is known analytically `ρ(x) = 1/(π√x(1-x))`. The test compares the probability of each bin using `transferoperator` and `invariantmeasure` with the probabilities obtained by integrating the analytical invariant measure over the same bins (10 bins were used).

`Intregrals` is used for the numerical integration. It doesn't appear anywhere in the source code, only in the unit test (notice that the dependency is only added to test/Project.toml ).